packages/lumx-core/src/scss/components/notification-block/lumapps/_index.scss

```diff
@@ -3,8 +3,19 @@
    ========================================================================== */
 
 .#{$lumx-base-prefix}-notification-block {
+    $self: &;
+
     display: flex;
     align-items: flex-start;
+    padding: $lumx-spacing-unit * 2 $lumx-spacing-unit * 3;
```

Comment:
You can use `$lumx-spacing-unit-big` and `$lumx-spacing-unit-huge` instead

Reply:
No padding please, padding should be implemented according context